### PR TITLE
#130341335 jasmine and karma setup

### PR DESCRIPTION
**Pull request by demiladebamgbose**
#### What does this PR do?
- Set up karma and jasmine for unit test for the front end
#### Description of Task to be completed?
- Change the test command
- Add simple tests to make sure the configuration works
#### How should this be manually tested?
- run `karma start --reporters kjhtml`
#### Any background context you want to provide?

The test suite used before was the mocha and now is being converted to Karma and Jasmine
#### What are the relevant pivotal tracker stories?

130341335

### CODE REVIEW

File: src/app/controllers/test-controller.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+angular.module('test', []).controller('TestController',
+  ['$scope', function($scope) {
+    $scope.countDown = function() {
+    if($scope.time) {
```
Comment (ruqoyyasadiq) [inline src/app/controllers/test-controller.js]: add indentation here

File: src/app/controllers/test-controller.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+      $scope.time -= 10;
+    }
+  };
+  $scope.time = 200;
```
Comment (ruqoyyasadiq) [inline src/app/controllers/test-controller.js]: move line 8 to line 3 i.e. define `scope.time` before using it in `scope.countDown`.

**Comment by ruqoyyasadiq** [comment]
- Fix houndCI's error
- follow the PR naming convention and add the appropriate description as per the directions in the wiki.

File: src/test/controllers/test-controller-spec.js [hunk 3]
```diff
@@ -9,7 +9,8 @@ describe('TestController', function() {
     inject(function($injector) {
       $rootScope = $injector.get('$rootScope');
       $scope = $rootScope.$new();
-      controller = $injector.get('$controller')('TestController', {$scope: $scope});
+      controller = $injector.get('$controller')('TestController',
+      {$scope: $scope});
```
Comment (ruqoyyasadiq) [inline src/test/controllers/test-controller-spec.js]: add one step indentation. When beaking a line of code, indent the next of break by one step so as to help know that it belongs to the previous line.

**Comment by ruqoyyasadiq** [comment]
Approved

File: src/test/controllers/test-controller-spec.js (hunk 3 above)
Comment (andela-rsadiq) [inline src/test/controllers/test-controller-spec.js]: Approved

**Review by ruqoyyasadiq — APPROVED** [review]

**Pull request merged**